### PR TITLE
Ebonilla/sy 751 clientpy time range write does not work

### PR DESCRIPTION
**Pull request by emilbon99**
# Fix Pull Request Template

## Key Information

- Linear Issue: [SY-751]:(https://linear.app/synnaxlabs/issue/SY-751/[clientpy]-time-range-write-does-not-work)

## Description

Fixed issue in documentation where it is not possible to write directly to a range.

## Basic Readiness Checklist

- [x] I have performed a self-review of my code.
- [x] I have added sufficient regression tests to cover the changes.

## Manual QA Additions

- [x] I have updated the [Release Candidate](/.github/PULL_REQUEST_TEMPLATE/rc.md) template
  with necessary manual QA steps to test my change.

### CODE REVIEW

File: client/py/tests/test_ranger.py [hunk 1]
```diff
@@ -226,5 +226,69 @@ def test_clear_active(self, client: sy.Synnax):
...
+            name=name,
+            time_range=start.span_range(4 * sy.TimeSpan.SECOND),
+        )
+        assert len(rng[data_ch.name]) == 4
+        assert rng[data_ch.name][0] == 1.0
+        assert rng[data_ch.name][1] == 2.0
+        assert rng[data_ch.name][2] == 3.0
+        assert rng[data_ch.name][3] == 4.0
+
+    @pytest.mark.focus
```
Comment (LeonLiur) [inline client/py/tests/test_ranger.py:266]: should remove mark.focus

File: client/py/tests/test_frame.py [hunk 2]
```diff
@@ -195,3 +195,33 @@ def test_adapation_of_channel_dict(self, adapter: [WriteFrameAdapter, sy.Channel
...
+        self,
+        adapter: [WriteFrameAdapter, sy.Channel],
+    ):
+        adapter, channel = adapter
+        o = adapter.adapt([channel], [1.0])
+        assert o.channels[0] == channel.key
+        assert o.series[0].data_type == sy.DataType.FLOAT64
+        assert o.series[0][0] == 1.0
+
+    @pytest.mark.focus
```
Comment (LeonLiur) [inline client/py/tests/test_frame.py:218]: Should remove focus

**Review by LeonLiur — APPROVED** [review]
Good once you address my comments

File: client/py/synnax/framer/adapter.py [hunk 3]
```diff
@@ -90,27 +88,28 @@ def __adapt_ch(
...
                 )
             if isinstance(series, list) and len(series) > 1:
-                raise ValidationError(
-                    f"""
-                Received a single channel {'name' if isinstance(channels_or_data, ChannelName) else 'key'}
-                but multiple series.
-                """
-                )
+                first = series[0]
+                if not isinstance(first, (float, int, TimeoutError)):
```
[the remark this answers is not in the document]
Comment (emilbon99) [reply client/py/synnax/framer/adapter.py]: We shouldn't. No idea how this ended up in there.

**Pull request merged**